cpp/ql/src/Likely Bugs/Likely Typos/UsingStrcpyAsBoolean.qhelp

```diff
@@ -21,8 +23,8 @@ and that do not have a return value reserved to indicate an error.</p>
   <li>_mbsncpy</li>
   <li>_mbsncpy_l</li>
 </ul>
```

Comment:
I think all of the function names in the list should be inside a code element (e.g. as strcpy_s is below).